docs/getting-started/installation.md

```diff
@@ -40,7 +40,9 @@ If you download or clone the repository, you must [build](../developers/contribu
 
 # Selecting the Correct Build
 
-Chart.js provides two different builds that are available for your use.
+Chart.js provides two builds: `Chart.js`, `Chart.min.js`. Both `Chart.js` and `Chart.min.js` files include Chart.js as well as the color parsing library. If this version is used, you are required to include [Moment.js](http://momentjs.com/) before Chart.js for the functionality of the time axis.
+
+Both `Chart.bundle.js` and `Chart.bundle.min.js` builds include Moment.js in a single file. You should use this version if you require time axes and want to include a single file. You should not use this build if your application already included Moment.js. Otherwise, Moment.js will be included twice which results in increasing page load time and possible version compatability issues.
```

Comment:
I think it would be better to split this text in both following sub-sections, replacing the existing text.

Reply:
For some reason, I wasn't aware of the sub-sections. The text is split into sub-sections in both .md. Thanks for pointing out:)